signac/contrib/project.py

```diff
@@ -51,36 +48,10 @@
 JOB_ID_LENGTH = 32
 JOB_ID_REGEX = re.compile(f"[a-f0-9]{{{JOB_ID_LENGTH}}}")
 
-# The warning used for doc filter deprecation everywhere. Don't use
-# triple-quoted multi-line string to avoid inserting newlines.
-# (issue #725) TODO: In signac 2.0, remove all docstrings for doc_filter parameters. The
-# doc_filter parameters will only be preserved for backwards compatibility but
-# not advertised as part of the API in signac 2.0.
-DOC_FILTER_WARNING = (
-    "The doc_filter argument is deprecated as of version 1.7 and will be removed "
-    "in version 3.0. Users should instead use a filter with a 'doc.' prefix. For "
```

Comment:
Needs a backport to `2.0` in the `master` branch.

Reply:
Good call. This is a tiny change so I'm just going to push it directly onto master.